### ISSUE TITLE
Simulation Experiment view: allow the plotting of 'advanced' data

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#113]
With COR and OpenCell, we can plot simple data, e.g. the L-type calcium current against time. However, we might want to plot the sum of the L-type and T-type calcium currents against time (as I believe can be done in JSim). To do this, we would currently have to create a new variable in the CellML file and then select it for output. This is clearly not ideal, hence this feature request. Note that such advanced data could be used for plotting either on the X or Y axis.

**Comment by nickerso** [comment]
This is also a requirement for SED-ML and data generators can be the arbitrary combination of multiple variables from a model. I don't think this should be handled by adding variables to the model.

**Comment by agarny** [comment]
Indeed, this shouldn't be handled by adding variables to a model. Yet, this is what we currently need to do if we want to achieve that result.

With regards to SED-ML, any 'advanced' data generated through the `Single Cell` view would indeed be exported to SED-ML.
